railties/lib/rails/generators/rails/app/templates/config/environments/development.rb.tt

```diff
@@ -15,8 +15,12 @@ Rails.application.configure do
   # Enable server timing.
   config.server_timing = true
 
-  # Enable/disable Action Controller caching. By default Action Controller caching is disabled.
-  # Run rails dev:cache to toggle Action Controller caching.
+  # Set canonical url for your application that'll be accessible via Rails.application.url.
+  # Used by Action Controller, Action Mailer, Active Storage for generating urls outside of active requests.
+  # config.application_url = "http://localhost:3000"
+
+  # Enable/disable caching. By default caching is disabled.
+  # Run rails dev:cache to toggle caching.
```

Comment:
This comment looks slightly different in the diff, maybe rebase issue?

Reply:
should be fixed